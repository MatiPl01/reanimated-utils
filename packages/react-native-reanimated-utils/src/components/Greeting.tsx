import { StyleSheet, Text, View } from 'react-native';

import { greet } from '../utils';

export default function Greeting() {
  return (
    <View style={styles.container}>
      <Text style={styles.text}>{greet()}</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    alignItems: 'center',
    backgroundColor: '#fff',
    flex: 1,
    justifyContent: 'center',
    padding: 16
  },
  text: {
    color: '#333',
    fontSize: 24
  }
});
