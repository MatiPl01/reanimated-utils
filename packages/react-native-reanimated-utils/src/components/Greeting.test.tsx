import { render } from '@testing-library/react-native';

import Greeting from './Greeting';

describe(Greeting, () => {
  it('renders', () => {
    const tree = render(<Greeting />);

    expect(tree).toMatchSnapshot();
  });
});
