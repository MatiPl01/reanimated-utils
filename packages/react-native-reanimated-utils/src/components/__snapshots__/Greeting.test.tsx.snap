// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting renders 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
      "padding": 16,
    }
  }
>
  <Text
    style={
      {
        "color": "#333",
        "fontSize": 24,
      }
    }
  >
    Hello from the library! ðŸ‘‹
  </Text>
</View>
`;
